pkg/vdri/bloc/peer/peer.go

```diff
@@ -0,0 +1,11 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package peer
+
+// Peer include info about peer
+type Peer struct {
```

Comment:
Rename to Endpoint (or something like that). Also need a domain name field.

Reply:
done